pages/blog/release-notes-2.5.0.md

```diff
@@ -0,0 +1,137 @@
+AsyncAPI 2.5.0 remedies this deficiency by allowing to use `Reference Object` whenever `Server Variable Object` can be used inside `Server.variables` field.
+
+Following example is now fully compliant with the specification:
+
+```yaml
+asyncapi: 2.5.0
+servers:
+  development:
+    $ref: '#/components/servers/myserver'
+  production:
+    $ref: '#/components/servers/myserver'
+components:
+  servers:
+    myserver:
+      url: "{stage}.my-server.com:{port}"
+      protocol: ws
+      variables:
+        stage:
+          $ref: "#/components/serverVariables/stage"
+        port:
+          $ref: "#/components/serverVariables/port"
+  serverVariables:
+    stage:
+      default: dev
+    port:
+      enum: [5000, 6000]
+      default: 5000
+```
+
+This new feature was contributed by [Vladimír Gorej](https://github.com/char0n). For more detail, see [Vladimír's `/spec #776` pull request](https://github.com/asyncapi/spec/pull/776) and the [GitHub issue where Vladimír's change to `serverVariables ` was discussed](https://github.com/asyncapi/spec/issues/775).
+
+## Decorating Server Object with tags
+
+Tags are used to categorize or group resources. 2.5.0 introduces support for defining tags at Server level, which can be used for many use cases like grouping servers by their features: environment name, cluster name, visibility, owner, etc.
+
+For example:
+
+```yaml
+asyncapi: '2.5.0'
+info:
+  title: Streetlights Kafka API
+  version: '1.0.0'
+servers:
+  scram-connections:
+    url: test.mykafkacluster.org:18092
+    protocol: kafka-secure
+    description: Test broker secured with scramSha256
+    security:
+      - saslScram: []
+    tags:
+      - name: "env:test-scram"
+        description: "This environment is meant for running internal tests through scramSha256"
+      - name: "kind:remote"
+        description: "This server is a remote server. Not exposed by the application"
+      - name: "visibility:private"
+        description: "This resource is private and only available to certain users"  
+  mtls-connections:
+    url: test.mykafkacluster.org:28092
+    protocol: kafka-secure
+    description: Test broker secured with X509
+    security:
+      - certs: []
+    tags:
+      - name: "env:test-mtls"
+        description: "This environment is meant for running internal tests through mtls"
+      - name: "kind:remote"
+        description: "This server is a remote server. Not exposed by the application"
+      - name: "visibility:private"
+        description: "This resource is private and only available to certain users"
+channels: {}
+```
+
+This new feature was contributed by [Sergio Moya](https://github.com/smoya). For more detail, see [Sergio's `/spec #809` pull request](https://github.com/asyncapi/spec/pull/809) and the [GitHub issue where Sergio's change was discussed](https://github.com/asyncapi/spec/issues/654).
+
+## Added missing IBMMQ Operation Binding
+
+content placeholder
```

Comment:
@magicmatatjahu can I ask to provide description of your contribution provided in https://github.com/asyncapi/spec/pull/840 and https://github.com/asyncapi/bindings/pull/149? Thanks a lot!

Reply:
I'm not a native speaker, but I would like to describe it in this way:

> We noted that in the case of bindings `ibmmq` we overlooked and did not define a definition for `Operation Bindings Object`. In new specification version the given definition is empty but to be consistent with the rest of the bindings it is reserved for future definition/values.

Could it be?

Reply:
Thanks. I've run it through the [Grammarly](https://grammarly.com/) which provided the final grammatically correct form. Here is the final result: https://github.com/asyncapi/website/pull/959/files#diff-1e152584ce16f037dccd187a75e11a4dfa94d6414ed40a90cbfdbff9af4cc210R101